src/Tribe/Tickets_Handler.php

```diff
@@ -807,7 +810,7 @@ public function get_post_totals( $post ) {
 			'has_unlimited' => false,
 			'has_shared'    => $global->is_enabled(),
 			'tickets'       => count( $tickets ),
-			'capacity'      => tribe_get_event_capacity( $post ),
+			'capacity'      => (int) tribe_get_event_capacity( $post ), // Could be null.
```

Comment:
This sounds out of scope of this ticket, we might want to be very wary about changes like this.

Reply:
```suggestion
			'capacity'      => tribe_get_event_capacity( $post ),
```

Reply:
@sc0ttkclark

iirc, https://github.com/moderntribe/event-tickets/blob/revert-1690-revert-1685-fix/ET-843-prevent-fatals-inactive-provider/src/Tribe/Admin/Columns/Tickets.php#L151 was throwing a notice and is the only spot this 'capacity' is used (via IDE autocomplete).

Searching through all our plugins for `['capacity']`, none such usages explicitly check for `null`, some for `''` and some cast to _int_ on their own.

I'd say with the thoroughness of the QA this ticket will receive, it's able to be included but obviously that's your call.